core/client/src/main/java/alluxio/client/file/RetryHandlingFileSystemWorkerClient.java

```diff
@@ -121,7 +125,7 @@ public void run() {
             } catch (InterruptedException e) {
               // do nothing
             } catch (Exception e) {
-              LOG.error("Failed to heartbeat for session {}", mSessionId, e);
+              LOG.warn("Failed to heartbeat for session {}", mSessionId, e);
```

Comment:
using a full stack trace here results in very verbose logging (as the initial retries happen rather quickly)